README.md

```diff
@@ -25,8 +25,9 @@ pip install terraform-local
 
 The following environment variables can be configured:
 * `TF_CMD`: Terraform command to call (default: `terraform`)
-* `LOCALSTACK_HOSTNAME`: host name of the target LocalStack instance
-* `EDGE_PORT`: port number of the target LocalStack instance
+* `LOCALSTACK_HOSTNAME`: __(Deprecated)__ host name of the target LocalStack instance
+* `EDGE_PORT`: __(Deprecated)__ port number of the target LocalStack instance
+* `AWS_ENDPOINT_URL`: hostname and port of the target LocalStack instance
```

Comment:
nit: perhaps we could move `AWS_ENDPOINT_URL` to the top (so users immediately notice it), and the deprecated configs below it (or even to the bottom of the list)..?